src/transforms/transform-pull-request.test.ts

```diff
@@ -629,6 +687,48 @@ describe("pull_request transform GraphQL", () => {
 		});
 	});
 
+	it("maps CHANGES_REQUESTED state to NEEDSWORK", async () => {
```

Comment:
/nit
looks like the description is repeated above?

Reply:
One is for testing the REST call and one is for testing the graphQL call